arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,6 +410,11 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The current state of Analysis Services resource. It allows more custom terminal state like Paused."
```

Comment:
@amarzavery @ravbhatnagar
The change from service side is after discussion with ARM architect, who has the final say on this.

'State' would be used for the state outside of resource provisioning
We would insist on 'State' as the ARM architect proposed it and renaming in service side would break ASPaaS.

Please let us know if any of you has further comment, thanks